lib/bump/bump.ts

```diff
@@ -127,6 +147,12 @@ export interface PullRequestOptions {
    * @default "master"
    */
   base?: string;
+
+  /**
+   * Set of labels to apply to the PR.
+   */
+  labels?: string[]
+
```

Comment:
`Labels` should be added on type `PullRequestOptions`  which is included in `AutoBumpOptions`

Reply:
Its is in `PullRequestOptions` - am I missing something?

Reply:
dammit, the diff view collapsed the `export interface PullRequestOptions` from the view  so it looked like it was added on `AutoBumpProps`, I will see myself out.